src/renderer/redux/selectors/discover.js

```diff
@@ -0,0 +1,35 @@
+import { createSelector } from 'reselect';
+import { selectFeaturedUris } from './content';
+import { selectClaimsById, selectAllClaimsByChannel } from './claims';
+
+export const selectDiscover = createSelector(
+  [selectFeaturedUris, selectClaimsById, selectAllClaimsByChannel],
+  (featuredUris, claimsById, claimsByChannel) => {
+    let categories = featuredUris;
+    if (!!categories && !!claimsByChannel) {
+      let channels = [];
+      Object.keys(categories).forEach(key => {
+        if (key.indexOf('@') === 0) {
+          channels.push(key);
+        }
+      });
+      Object.keys(claimsByChannel).forEach(key => {
+        if (channels.includes(key)) {
+          delete categories[key];
+          const ids = claimsByChannel[key][1];
+          let uris = [];
+          const newKey = `${key}#${claimsById[ids[0]].value.publisherSignature.certificateId}`;
```

Comment:
instead of using string interpolation, the proper way to do with is with the `buildURI` function

(I know I told you this method was fine earlier, sorry about that)